src/booking/booking.module.ts

```diff
@@ -7,6 +7,7 @@ import { NgModule } from '@angular/core';
 import { AlertService } from '@hmcts/ccd-case-ui-toolkit';
 import { SharedModule } from '../app/shared/shared.module';
 import { bookingRouting } from './booking.routes';
+
```

Comment:
Could be reordered to prevent the need for spaces however we agreed it would look a bit cramped and it sometimes looks more readable if we space them out a little.